### DISCUSSION TITLE
How to share a recipe?

### DISCUSSION THREAD

**Discussion by WeT-Klb** [mealie-recipes/mealie#977] [Q&A]
Hello,

when i right-click on one recipe inside Mealie there´s one item "Share" - but nothing happens when clicking it.
So how i can share it? How to invite somebody to watch the recipe (not invite someone to become a user)?
Yours
WeT-Klb

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened, so I'm going to close this. If you're still having issues with the sharing functionality feel free to open a new issue
